Advanced features (#2)
Serial-Link/CPP-Unix-Bindings: README.md

> # cpp_unix_bindings

Reviewer: ```suggestion
# C++ Bindings Linux
```

A compact C++23 library for talking to serial devices on Linux (e.g. Arduino).  

> The project builds a **shared library `libcpp_unix_bindings.so`** that can be used via

Reviewer: ```suggestion
The project builds a **shared library `bindings_linux.so`** that can be used via
```

cmake --build build -j

# The resulting library will be located at

> #   build/libcpp_unix_bindings.so

Reviewer: ```suggestion
#   build/bindings_linux.so
```

```ts
// serial_deno.ts

> const lib = Deno.dlopen('./build/libcpp_unix_bindings.so', {

Reviewer: ```suggestion
const lib = Deno.dlopen('./build/bindings_linux.so', {
```

Author: all libs under linux are prefixed with a `lib`, would not go against the default here

Author: go under `/usr/lib` or `/usr/lib64` in any linux distro and show me one lib that has not the prefix `lib` and is not built by you.
